docs/src/main/asciidoc/logging.adoc

```diff
@@ -571,12 +564,13 @@ test {
 
 See also <<getting-started-testing.adoc#test-from-ide,Running `@QuarkusTest` from an IDE>>.
 
-[[logging-adapters]]
-== Logging adapters
+[[logging-apis]]
+== Logging APIs and adapters
```

Comment:
The other headings were changed to imperative verb phrases; should this one be an imperative verb phrase too?

Reply:
Dmaned... forgot that one.